Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -308,6 +308,10 @@ define([
 
         if (gltfFormat === 0) {
             var gltfUrl = getStringFromTypedArray(gltfView);
+
+            // We need to remove padding from the end of the model URL in case this tile was part of a composite tile
+            // This removes all white space and null characters from the end of the string.
+            gltfUrl = gltfUrl.trim().replace(/\0+$/, '');
```

Comment:
OK, but if there's whitespace followed by null followed by more whitespace, this will fail right?  Should you get rid of `trim` and have a regex that removes all null and whitespace at the end?